package.json

```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
```

Comment:
for the future, don't add json files and other unnecessary files during the push of your files, push only src/. files and readme.md, files in which you write code manually